RapidReact/src/main/java/frc/robot/Robot.java

```diff
@@ -338,41 +344,32 @@ private void configureButtonBindings() {
         )
         .whenReleased(() -> driverClimbEnabledPressed = false);
 
-      buttons.elevatorExtend.whenPressed(climberSubsystem::elevatorExtend);
+      buttons.elevatorExtend.whenPressed(climberSubsystem::manualElevatorExtend);
       buttons.elevatorExtend.whenReleased(climberSubsystem::elevatorStop);
 
-      buttons.elevatorRetract.whenPressed(climberSubsystem::elevatorRetract);
+      buttons.elevatorRetract.whenPressed(climberSubsystem::manualElevatorRetract);
       buttons.elevatorRetract.whenReleased(climberSubsystem::elevatorStop);
-
-      buttons.climbAuto.whenPressed(
-        () -> {
-          climberSubsystem.climbAuto();
-          rgbSubsystem.autoClimb();
-        }
-      );
+      
+      buttons.climbAuto.whenPressed(climberSubsystem::autoClimb);
+      buttons.climbAuto.whenReleased(climberSubsystem::autoClimbReleased);
+      buttons.resetClimbStuff.whenPressed(climberSubsystem::resetClimbStuff);
     }
-
+    
     //Shooter BUttons and Climber Buttons
-    if (config.enableClimberSubsystem || config.enableShooterSubsystem) {
-      buttons.hubShootOrFixedHookToggle.whenPressed(
-        () -> {
-          if (config.enableClimberSubsystem && climberEnabled) {
-            climberSubsystem.fixedHookToggler();
-          } else {
-            if (config.enableShooterSubsystem) {
-              shooterSubsystem.shootTop();
-            }
-          }
+    if (config.enableClimberSubsystem || config.enableShooterSubsystem)
```

Comment:
This doesn't seem to have anything to do with the climber. Fix this if statement.